src/Components/List.jsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { useLocation, Redirect } from 'react-router-dom';
 
-export default function List() {
-  return <div>Hi from List View!</div>;
+export default function List({ token }) {
+  let location = useLocation().pathname;
+
+  return (
+    <div>
+      {token === null && location === '/list' ? (
+        <Redirect to="/" />
+      ) : (
+        <div>Hi from List View!</div>
+      )}
+    </div>
```

Comment:
Same comments as the ones left on the `AddItem.jsx` file.

Reply:
Same comments from AddItem